locale/fi/panel.json

```diff
@@ -53,7 +53,7 @@
         {
           "image.key": "",
           "image.alt": "",
-          "text": "Käytä laajennuspanelia ainoastaan sisällölle jonka ei ole välttämätöntä olla käyttäjälle näkyvissä koko ajan. Laajennuspanelia voidaan myös käyttää vähäisemmän prioriteetin sisällölle.\n"
+          "text": "Käytä laajennuspanelia ainoastaan sisällölle jonka ei ole välttämätöntä olla käyttäjälle näkyvissä koko ajan. Laajennuspanelia voidaan myös käyttää vähäisemmän prioriteetin sisällölle."
```

Comment:
Osassa sisällöistä puhutaan laajennuspanelista (yhdellä e:llä) ja osassa laajennuspaneelista. Jälkimmäinen on oikein. Jos on järkevää fiksata tässä niin kannattaa tehdä samoilla lämpimillä.

Reply:
Fixed all to follow the latter spelling.